docs/products/postgresql/howto/identify-pg-slow-queries.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Identify PostgreSQL® slow queries
+title: Identify PostgreSQL® slow queries with `pg_stat_statements`
```

Comment:
"\`pg_stat_statements\`" doesn't render as a code but retains the back ticks in the preview.